swagger.yaml

```diff
@@ -2,53 +2,148 @@ swagger: "2.0"
 info:
   description: "An API used to query information about datasets published by the ONS."
   version: "1.0.0"
-  title: "ONS Dataset API"
+  title: "Dataset API"
   license:
     name: "Open Government Licence v3.0"
     url: "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
 basePath: "/v1"
 tags:
-- name: "Dataset"
+- name: "Public"
+- name: "Private user"
+- name: "Private"
 schemes:
 - "http"
 parameters:
-  id:
-    name: id
-    description: "Id that represents a dataset type"
+  dimension:
+    name: dimension
+    description: "A dimension from a dataset"
     in: path
-    type: string
     required: true
+    type: string
   edition:
     name: edition
     description: "An edition of a dataset"
     in: path
-    type: string
     required: true
-  version:
-    name: version
-    description: "A version of a dataset"
+    type: string
+  event:
+    name: event
+    description: "An import event when publishing a dataset"
+    in: body
+    schema:
+      $ref: '#/definitions/Event'
+  id:
+    name: id
+    description: "Id that represents a dataset type"
     in: path
+    required: true
     type: string
+  inserted_observations:
+    name: inserted_observations
+    description: "A value to increment the inserted_observations within an instance"
+    in: path
     required: true
-  dimension:
-    name: dimension
-    description: "A dimension from a dataset"
+    type: integer
+  instance_id:
+    name: instance_id
+    description: "A unique id for an instance"
     in: path
+    required: true
     type: string
+  instance:
+    name: instance
+    description: "A state of an instance"
+    in: body
+    required: true
+    schema:
+      $ref: '#/definitions/Instance'
+  import_job:
```

Comment:
no longer required?

Reply:
Its is still needed to create a link between a job and an instance